eth/handler.go

```diff
@@ -198,10 +198,10 @@ func newHandler(config *handlerConfig) (*handler, error) {
 		// the propagated block if the head is too old. Unfortunately there is a corner
 		// case when starting new networks, where the genesis might be ancient (0 unix)
 		// which would prevent full nodes from accepting it.
-		if h.chain.CurrentBlock().NumberU64() < h.checkpointNumber {
-			log.Warn("Unsynced yet, discarded propagated block", "number", blocks[0].Number(), "hash", blocks[0].Hash())
-			return 0, nil
-		}
+		// if h.chain.CurrentBlock().NumberU64() < h.checkpointNumber {
```

Comment:
I just mean, if we do not want this code enabled, it should be deleted instead of commented out. Leaving a bunch of commented code blocks results in cruft collecting in the codebase

Reply:
Issue relating to the code being removed: https://github.com/spruce-solutions/go-quai/issues/302